Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -634,6 +652,31 @@ theorem rpow_nat_inv_pow_nat {x : ℝ} (hx : 0 ≤ x) {n : ℕ} (hn : n ≠ 0) :
   rw [← rpow_nat_cast, ← rpow_mul hx, inv_mul_cancel hn0, rpow_one]
 #align real.rpow_nat_inv_pow_nat Real.rpow_nat_inv_pow_nat
 
+lemma rpow_strictMono_of_base_gt_one {b : ℝ} (hb : 1 < b) :
```

Comment:
```suggestion
lemma strictMono_rpow_of_one_lt_base {b : ℝ} (hb : 1 < b) :
```
to follow the order of symbols in the statement of the lemma. Same thing for the next lemmas.

Reply:
Changed.